src/main/java/core/mvc/DispatcherServlet.java

```diff
@@ -38,26 +41,34 @@ public void init() {
         handlerAdapterRegistry.addHandlerAdapter(new ControllerHandlerAdapter());
 
         handlerExecutor = new HandlerExecutor(handlerAdapterRegistry);
+
+        interceptorRegistry = new InterceptorRegistry();
+        interceptorRegistry.addInterceptor("/api/*", new ProcessingTimeCheckInterceptor());
```

Comment:
```suggestion
        interceptorRegistry = new InterceptorRegistry();
        
        InterceptorRegistration interceptorRegistration = new InterceptorRegistration()
                .addInterceptor(new ProcessingTimeCheckInterceptor())
                .addPathPatterns("/api/*");

        interceptorRegistry.addInterceptor(interceptorRegistration);
```
예시처럼 Interceptor의 사용 방식을 정의 할 수 있는 설계 방법을 고민해본다면
더 유연하게 Interceptor를 사용 할 수 있을거라고 생각해요. 😄